test/configs/autoconfig_test.sh

```diff
@@ -30,9 +30,18 @@ WORKSPACE_ROOT=$(pwd)
 TARGET=${TEST_BINARY%_test}
 NAME=${TARGET##*/}
 DIR=${TARGET%${NAME}}
-
 autoconfig_script=${WORKSPACE_ROOT}/${DIR}${NAME}
 
+# Helper function for always delete the containers / temporary files on exit
+function cleanup_on_finish {
+  echo "=== Deleting images  ==="
+  docker images -a | grep "bazel/${DIR%/}" | awk '{print $3}' | xargs docker rmi -f
+  docker images -a | grep "test-" | awk '{print $3}' | xargs docker rmi -f
```

Comment:
For name-match in `docker ps` and deletion, we can do:
```
docker rm -f $(docker ps -f name=test- -q)
```

Reply:
The containers have already been deleted after the docker run command by using the --rm flag. I'm also sending a PR https://github.com/GoogleCloudPlatform/base-images-docker/pull/209 to delete container after install_pkgs rule. We are removing the images here.

Reply:
I see, it looks like `docker images -f` doesn't support `name` filter, so LGTM then.